freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -149,3 +158,39 @@ def _filter_bundles_by_pinned_related_images(self, bundle_image_nvrs):
                     continue
                 ret_bundle_images_nvrs.add(nvr)
         return ret_bundle_images_nvrs
+
+    def get_published_original_nvr(self, session, rebuilt_nvr):
+        """
+        Search for an original build, that has been built and published to a
+            repository, and get original_nvr from it
+
+        :param session: database session
+        :param str rebuilt_nvr: rebuilt NVR to look build by
+        :rtype: str or None
+        :return: original NVR from the first published FM build for given NVR
+        """
+        original_nvr = None
+        # build event should be only one in database, or raise an error
+        build_event = db.session.query(ArtifactBuild).filter(
```

Comment:
`build_event` is misleading. It sounds like a Freshmaker event but it's not. Could you please use `artifact_build` or something like that?